### PR TITLE
Bztools nosetest b2g query creator

### PR DESCRIPTION
**Pull request by anoopvalluthadam**
Sorry for the new Pull request, since I deleted the branch by mistake!

### CODE REVIEW

**Comment by anoopvalluthadam** [comment]
@sylvestre qualified for merge? :)

**Comment by anoopvalluthadam** [comment]
@sylvestre

File: auto_nag/tests/test_phonebook.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+from auto_nag.scripts.phonebook import PhonebookDirectory
+
+
+class TestPhonebook:
+    def __init__(self):
+        self.demo_dict = {u'email@example.com': {
```
Comment (sylvestre) [inline auto_nag/tests/test_phonebook.py]: why don't you use the sample file?

Comment (anoopvalluthadam) [inline auto_nag/tests/test_phonebook.py]: I could use that, I thought, if something changed there(people.json), we are making sure that this is the format we are expecting.

**Comment by anoopvalluthadam** [comment]
Sorry for the mess up, Creating a new pull request!

**Pull request closed without merging**